### PR TITLE
Feature/414/calendar filter

### PR DESCRIPTION
**Pull request by kdanilenka32**
image

### CODE REVIEW

File: src/components/FormComponents/SelectController/SelectController.tsx [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
   name,
   control,
   options,
+  value,
   customChange,
   ...props
 }: SelectControllerProps<T>) => {
   const { t } = useTranslation('app');
 
+  const renderSelect = (
```
Comment (wildkiomi) [inline src/components/FormComponents/SelectController/SelectController.tsx:21]:
```suggestion
  const renderSelect = (
    onChange?: ((e: SelectEvent) => void),
    value?: string,
```

File: src/features/Applet/Schedule/Filter/ExpandedList/ExpandedList.types.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+type Buttom = {
```
Comment (wildkiomi) [inline src/features/Applet/Schedule/Filter/ExpandedList/ExpandedList.types.ts:1]:
```suggestion
type Button = {
```

File: src/features/Applet/Schedule/Filter/Filter.const.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    value: 'Default Schedule',
+    icon: <Svg id="calendar" />,
+  },
+  {
+    labelKey: 'individualSchedule',
+    value: 'Individual Schedule',
+    icon: <Svg id="user-calendar" />,
+  },
+];
+// TODO check trnslations after api connect
```
Comment (wildkiomi) [inline src/features/Applet/Schedule/Filter/Filter.const.tsx:32]:
```suggestion
// TODO check translations after api connect
```

File: src/svgSprite.ts [hunk 4]
```diff
@@ -554,5 +554,22 @@ export const svgSprite = (): string => `
...
+    <path d="M3.14737 14.95C2.75685 15.3405 2.75687 15.9737 3.1474 16.3642L5.70782 18.9245L3.14838 21.4839C2.75785 21.8744 2.75783 22.5076 3.14834 22.8981C3.53886 23.2886 4.17202 23.2886 4.56256 22.8981L7.12207 20.3387L9.68158 22.8981C10.0721 23.2886 10.7053 23.2886 11.0958 22.8981C11.4863 22.5076 11.4863 21.8744 11.0958 21.4839L8.53632 18.9245L11.0967 16.3642C11.4873 15.9737 11.4873 15.3405 11.0968 14.95C10.7063 14.5595 10.0731 14.5594 9.68256 14.95L7.12207 17.5103L4.56158 14.95C4.17105 14.5594 3.53788 14.5595 3.14737 14.95Z" />
+  </symbol>
+
+  <symbol viewBox="0 0 24 24" id="svg-export2">
```
Comment (wildkiomi) [inline src/svgSprite.ts:567]: maybe it's better to rename?

File: src/features/Applet/Schedule/Filter/ExpandedList/ExpandedList.types.ts (hunk 2 above)
Comment (IhorNesterenko99) [inline src/features/Applet/Schedule/Filter/ExpandedList/ExpandedList.types.ts:1]: Pls fix the typo (Button)

File: src/features/Applet/Schedule/Filter/Filter.const.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import { Fragment } from 'react';
+import i18n from 'i18n';
```
Comment (IhorNesterenko99) [inline src/features/Applet/Schedule/Filter/Filter.const.tsx:2]: Is this an internal import? Pls check spacing.

File: src/features/Applet/Schedule/Filter/Filter.utils.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+import { Box } from '@mui/system';
+import i18n from 'i18n';
```
Comment (IhorNesterenko99) [inline src/features/Applet/Schedule/Filter/Filter.utils.tsx:3]: pls check spacing

File: src/styles/variables/index.ts [hunk 7]
```diff
@@ -22,6 +22,7 @@ export const variables = {
   },
   borderRadius: {
     half: '50%',
+    xxxl2: '99.9rem', //999px
```
Comment (IhorNesterenko99) [inline src/styles/variables/index.ts:25]: Do we need this border radius? I don't see it in the code.

File: src/components/FormComponents/SelectController/SelectController.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/components/FormComponents/SelectController/SelectController.tsx:21]: works only with value

File: src/svgSprite.ts (hunk 4 above)
Comment (kdanilenka32) [reply src/svgSprite.ts:567]: both icons for export what name better to use

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**